source/agora/node/Ledger.d

```diff
@@ -228,55 +230,69 @@ public class Ledger
 
     ***************************************************************************/
 
-    public void prepareNominatingSet (ref Set!Transaction txs) @safe
+    public void prepareNominatingSet (ref ConsensusData data) @safe
     {
-        assert(txs.length == 0);
+        assert(data.tx_set.length == 0);
 
         if (this.pool.length < Block.TxsInBlock)
             return;
 
         const ulong next_height = this.getBlockHeight() + 1;
         auto utxo_finder = this.utxo_set.getUTXOFinder();
 
+        // this.enroll_man.getUnregistered(data.enrolls);
```

Comment:
You will need to fix this too. you need to add the enrollments to the nominating set.

Reply:
It's been closed for a while for the test.